src/requestManager.cpp

```diff
@@ -522,39 +539,61 @@ void CRequestManager::SendRequests()
 		    {
                     if (1)
                       {
-                      cs_objDownloader.unlock();
-                      LOCK(next.node->cs_vSend);
-  		      // from->AskFor(item.obj); basically just shoves the req into mapAskFor
-                      if (!item.lastRequestTime || (item.lastRequestTime && IsChainNearlySyncd()))
-                        {
-                          next.node->mapAskFor.insert(std::make_pair(now, item.obj));
-                          item.outstandingReqs++;
-		          item.lastRequestTime = now;
-                        }
-                      cs_objDownloader.lock();
-		      }
-                    if (0)  // SAME NODE RETRY DISABLED: Won't help in a TCP connection anyway: 
```

Comment:
I'm not sure I agree with leaving our thought processes in the main code
branch, it makes the code more difficult to read through a complex
section of code IMO.  Other than //TODO comments, I believe those are
things we should have in our own private branches that we're working
on.  But it's up to you, i'm not going to make a big thing out of it...

.On 04/09/2016 3:48 AM, gandrewstone wrote:
>
> In src/requestManager.c
> <https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/79#discussion_r77450382>
>
> > @@ -522,39 +539,61 @@ void CRequestManager::SendRequests()
> >  		    {
> >                      if (1)
> >                        {
> > -                      cs_objDownloader.unlock();
> > -                      LOCK(next.node->cs_vSend);
> > -  		      // from->AskFor(item.obj); basically just shoves the req into mapAskFor
> > -                      if (!item.lastRequestTime || (item.lastRequestTime && IsChainNearlySyncd()))
> > -                        {
> > -                          next.node->mapAskFor.insert(std::make_pair(now, item.obj));
> > -                          item.outstandingReqs++;
> > -		          item.lastRequestTime = now;
> > -                        }
> > -                      cs_objDownloader.lock();
> > -		      }
> > -                    if (0)  // SAME NODE RETRY DISABLED: Won't help in a TCP connection anyway: 
>
> WRT removal of "if (0)" and subsequent logic: I want to preserve my
> thought process here and keep the code logically correct in case we
> move to multiple transports...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/79/files/efaaa2474cc017ebc1c84fa3748b50c293bff5c7#r77450382>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMRF0AKDSr0sDaLrSA5Q8n1in31fPA3oks5qmqHqgaJpZM4JokbP>.
>